DAS/das/tests/fake_module_tester.py

```diff
@@ -163,22 +164,22 @@ def publish(client, topic, data={}):
     """
     Publishes python dict data to a specific topic in JSON and prints it out
     client: MQTT client object
-    topic:  MQTT topic eg. '/v3/wireless-module/<id>/start'
+    topic:  MQTT topic eg. '/v3/wireless_module/<id>/start'
     data:   Python dict containing the data to be published on the topic
     """
     # Generate JSON from the python dict
     json_data = json.dumps(data)
 
     # Publish the data over MQTT
-    client.publish(topic, json_data)
+    client.publish(str(topic), json_data)
     print(topic, "--> ", json_data)
 
 
 def start_modules(args):
     """ Sends a null message on the start channels for all of the selected
```

Comment:
Needs imperative mood here.

Reply:
Good catch! However, it seems that much of the documentation in these files (unchanged in this PR) aren't written in imperative mood, so I reckon it's possibly best left for another PR. @Blake-Haydon, perhaps this could be sneaked into your existing documentation PR? I promise to give you a speedy review.